Reference/Cache/index.md

```diff
@@ -61,12 +62,14 @@ When an `ICacheRefresher` is executed via the `DistributedCache.Instance` a noti
 
 ## Events handling to refresh cache
 
-TODO: fill in these docs(https://github.com/umbraco/UmbracoDocs/issues/208)
+To use the extensions add a using to `Umbraco.Web.Cache`;  You can then call them on the DistributedCache.Instance object.
```

Comment:
call them or invoke them?

an example here of using the extensions?

Reply:
Examples are a good idea, but can be added later IMHO